build.gradle.kts

```
@@ -71,10 +71,10 @@ val enableSpotBugs = props.bool("spotbugs")
 val enableCheckerframework by props()
 val enableErrorprone by props()
 val enableDependencyAnalysis by props()
-val enableParameters by props(true)
 val skipJandex by props()
 val skipCheckstyle by props()
 val skipAutostyle by props()
+val skipJavacParameterNames by props(true)
```

I would go with adding parameter names by default.

```suggestion
val skipJavacParameterNames by props()
```